Update README.md

digismash opened the pull request:
A brief description of the changes in this PR:


Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/zwalletcli/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

Associated PRs (Link as appropriate):
- blobber:
- gosdk:
- system_test:
- zboxcli:
- 0chain:
- Other: ...

digismash committed:
Update README.md

digismash committed:
Update README.md

digismash committed:
Update README.md

digismash committed:
zwalletcli cleanup README.md

digismash committed:
zwalletcli cleanup README.md

HassanAbbas7357 commented on README.md:
```diff
@@ -1,6 +1,6 @@
 # zwallet - a CLI for Züs wallet
 
-`zwallet` is a command line interface (CLI) to demonstrate the wallet functionalities of Züs.
+ZWallet CLI is a Go-based command-line tool for managing ZCN tokens on the Züs blockchain. It provides an interactive way to receive, send, store, stake, and exchange ZCN tokens, as well as fetch information about Züs components such as miners, sharders, and blobbers.
```
Missing: ZWallet CLI is also used to create and manage wallet

HassanAbbas7357 commented on README.md:
```diff
@@ -44,23 +44,23 @@ The CLI utilizes the [Züs GoSDK](https://github.com/0chain/gosdk).
+[Züs](https://zus.network/) is a high-performance cloud on a fast blockchain offering privacy and configurable uptime. It is an alternative to traditional cloud S3 and has shown better performance on a test network due to its parallel data architecture. The technology uses erasure code to distribute the data between data and parity servers. Züs storage is configurable to provide flexibility for I ...
 
-For instance, the user can start with 10 data and 5 parity providers and select where they are located globally, and later decide to add a provider on-the-fly to increase resilience, performance, or switch to a lower cost provider.
+For instance, the user can start with 10 data and 5 parity providers and select where they are located globally, and later decide to add a provider on-the-fly to increase resilience, and performance or switch to a lower-cost provider.
```
Lower-cost provider is fine but Low-cost provider is most commonly used phrase

HassanAbbas7357 commented on README.md:
```diff
@@ -109,20 +109,19 @@ For detailed steps on the installation, follow the guides below:
-| `--configDir` | Config directory                | `~/.zcn`       |
-| `--network`   | [Network file](#override-network) | `network.yaml` |
-| `--silent`    | Do not print detailed logs      | `false`        |
+| `--h,--help`  | Shows help/parameters for a particular command (./zwallet -h) will list all help commands. To know detail about a particular command like example 'send' use (./zwallet send --help )                     |                |
```
for --help command spacing is not correct

HassanAbbas7357 commented on README.md:
```diff
@@ -109,20 +109,19 @@ For detailed steps on the installation, follow the guides below:
-| `--silent`    | Do not print detailed logs      | `false`        |
+| `--h,--help`  | Shows help/parameters for a particular command (./zwallet -h) will list all help commands. To know detail about a particular command like example 'send' use (./zwallet send --help )                     |                |
+| `--config`    |  [Config file](https://github.com/0chain/zwalletcli/blob/staging/network/config.yaml) and [description](https://github.com/0chain/zwalletcli#zcnconfigyaml). Configuration file    | `config.yaml`  |
+| `--configDir` | Configuration directory                | `~/.zcn`       |
```
incrase the parameter coplumn size so that commands can fit in and does not break line for commands

HassanAbbas7357 commented on README.md:
````diff
@@ -154,12 +153,13 @@ wallet saved in /home/ubuntu/.zcn/wallet.json
 ./zwallet faucet --methodName pour --input "new wallet"
 ```
-
+#### Creating a second wallet with 'faucet' command
````
The output for this command shows that this command is finding if a  wallet exists or not then its creating another wallet so its better to allign the command with the heading or change the heading to "Creating a second wallet with 'faucent' command " then its output should not show that the no wallet exists in the path because its task is to create the wallet not to find an existing wallet

HassanAbbas7357 commented on README.md:
```diff
@@ -667,7 +671,7 @@ To use a different wallet as sender, use `--wallet` global parameter.
 | `--hash`  | Yes      | Hash of transaction to verify |         | valid transaction hash |
 
-Note: Not all `zwallet` commands (eg. `send`) prints the transaction hash created. To see more details printed including the hashes, use `--verbose` global parameter.
+Note: Not all `zwallet` commands (e.g., `send`) prints the transaction hash created. To see more details printed, including the hashes, use `--verbose` global parameter.
```
(e.g., `send`) should be (e.g, `send`)

HassanAbbas7357 reviewed (changes requested):
I have commented out some minor mistakes please walk though the comments to fix them accordingly

digismash committed:
zwalletcli review comments and cleanup Update README.md

digismash committed:
zwalletcli cleanup - Update README.md

example outputs of each option of each command for most cases for better understanding.

digismash committed:
Clean up of zwallet commands and responses Update README.md

digismash committed:
zwallet cli clean up and details Update README.md

digismash committed:
zwallet cli cleanup with additional responses Update README.md

hm90121 committed:
add sc owner commands

hm90121 committed:
add easy to use installation instructions

hm90121 committed:
add xcode instructions

hm90121 committed:
correct typos

hm90121 committed:
fix renders

hm90121 committed:
typos and overide network steps
